apps/sample-app/src/components/TokenFetch.tsx

```diff
@@ -0,0 +1,41 @@
+import { useMsal } from '@azure/msal-react';
+import { Spinner, SpinnerSize } from '@fluentui/react';
+import { authentication } from '@microsoft/teams-js';
+import React from 'react';
+
+import { loginRequest } from './authConfig';
+
+const TokenFetchComponent: React.FC = () => {
+  const { instance, accounts } = useMsal();
+
+  const [accessToken, setAccessToken] = React.useState<string>();
+
+  React.useEffect(() => {
+    const request = {
+      ...loginRequest,
+      account: accounts[0],
+    };
+    instance.acquireTokenSilent(request).then(response => {
+      setAccessToken(response.accessToken);
+      authentication.notifySuccess(response.accessToken);
+    });
+    instance.acquireTokenSilent(request).catch(response => {
```

Comment:
You most likely don't want to call this again, instead you should attach the `catch()` to the initial `acquireTokenSilent` promise:

```
instance.acquireTokenSilent(request).then(response => {
      setAccessToken(response.accessToken);
      authentication.notifySuccess(response.accessToken);
    }).catch(response => { ... });
```